src/Layout/Presentation/SingleDistrict/InfoBox.tsx

```diff
@@ -1,13 +1,16 @@
 import * as React from "react";
 import { VulnerableDistrictSeat, VulnerableVotes } from "../../../utilities/district";
+import { Dictionary } from "utilities/dictionary";
```

Comment:
Codacy found an issue: [Submodule import paths from this package are disallowed; import from the root instead](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5947793)